metricbeat/helper/server/http/http.go

```diff
@@ -130,6 +148,11 @@ func (h *HttpServer) handleFunc(writer http.ResponseWriter, req *http.Request) {
 
 	case "GET":
 		writer.WriteHeader(http.StatusOK)
-		writer.Write([]byte("HTTP Server accepts data via POST"))
+		if h.server.TLSConfig != nil && h.server.TLSConfig.Certificates != nil {
```

Comment:
The request itself can tell you if it was received over TLS: `if req.TLS != nil`. Sorry to bother you about this line again, but the thought came to be late.

Reply:
No problem. It looks better this way. Let me test it...

Reply:
@andrewkroh Done, tested ok